Formula/jdtls.rb

```diff
@@ -0,0 +1,32 @@
+class Jdtls < Formula
+  desc "Java language specific implementation of the Language Server Protocol"
+  homepage "https://github.com/eclipse/eclipse.jdt.ls"
+  url "https://download.eclipse.org/jdtls/milestones/1.7.0/jdt-language-server-1.7.0-202112161541.tar.gz"
+  sha256 "2f0c28dfec317a268ec44904420657181b43a7ba2a32f0bf788ea388dacb8552"
+  license "EPL-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    libexec.install "config_mac"
+    libexec.install "features"
+    libexec.install "plugins"
+
+    (buildpath/"jdtls").write <<~BASH
+      #!/bin/bash
+      # copy the configuration folder to tmp to be writable
+      tmp_dir="$(mktemp -d /tmp/jdtls.XXXXX)"
+      cp -R #{libexec}/config_mac "${tmp_dir}"
+      # and ensure that it is removed on exit
+      trap "{ rm -rf ${tmp_dir}; }" EXIT
```

Comment:
This should just use `#{etc}`, which is writable.

Reply:
I'll still have to use the temporary directory because the test fails that I'm working on. `jdtls` tells me that `#{etc}/jdtls` must be writable which isn't the case during tests.

Reply:
Why not just use commands that don't write anything?

Reply:
I don't know how it could be solved with brew (its my first brew formula experience). `jdtls` requires a configuration directory that is writable and [the AUR package](https://aur.archlinux.org/cgit/aur.git/tree/launcher.sh?h=jdtls) and the [jdtls-launche](https://github.com/eruizc-dev/jdtls-launcher/blob/master/jdtls-launcher.sh) ensure it somehow.

Reply:
See [here](https://github.com/Homebrew/homebrew-core/pull/92550#discussion_r786737765). Resolving issue.